src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/ProjectExtensions.cs

```diff
@@ -7,18 +7,8 @@ namespace Microsoft.CodeAnalysis.Razor.Workspaces.Extensions
 {
     internal static class ProjectExtensions
     {
-        internal static Document GetRequiredDocument(this Project project, DocumentId? documentId)
```

Comment:
Lol, this was a bit silly before.

Reply:
The other benefit of this change, with the new syntax the compiler will warn about such silliness.